autoload/airline/extensions/ctrlspace.vim

```diff
@@ -5,10 +5,11 @@ scriptencoding utf-8
 
 let s:spc = g:airline_symbols.space
 let s:padding = s:spc . s:spc . s:spc
+let s:cs = ctrlspace#context#Configuration().Symbols.CS
 
 function! airline#extensions#ctrlspace#statusline(...)
   let b = airline#builder#new({ 'active': 1 })
-  call b.add_section('airline_b', '⌗' . s:padding . ctrlspace#api#StatuslineModeSegment(s:padding))
+  call b.add_section('airline_b', s:cs . s:padding . ctrlspace#api#StatuslineModeSegment(s:padding))
```

Comment:
Is it a problem, if users change the CS symbol later on and it won't be shown then?

Reply:
It will show the '#', the default CS symbol.
But if the font doesn't support the default CS symbox, it will be '?' symbol in gvim.

Reply:
I understand what it is doing. However if the user later on changes the CS-symbol, it won't be picked up by airline anymore.

Reply:
Maybe I don't unstandard your first question and give a unclear reply.
And what I want to express is that:

Assuming my font used in vim is not supported fot  the default CS symbol '⌗'. So I add the following code in my vimrc to change the ctrlspace CS symbol as an example:
```
let g:CtrlSpaceSymbols = { "CS": "n"}
```
Now:
 - In default ctrlspace extension of airline, it would show a '?' symbol for the font not supporting '⌗'. 
 - And if add this fix to ctrlspace extension of airline, it would show 'n' symbol changed by the example code.

Reply:
yes I understand that. So you change g:CtrlSpaceSymbols in your .vimrc. Now what happens later in your current session, if you change the Symbol again, e.g. after startup you issue `:let g:CtrlSpaceSymbols = { "CS": "#"}`? vim-airline won't notice, that you have changed it and will still use the old symbole, doesn't it?

Reply:
I have tried the situation you say just now. And it's just going on as what you say.
But if I change the g:CtrlSpaceSymlobs again after startup, vim-ctrlspace will noitce it too.
And in my gvim, though vim-ariline will still use the old CS symbol that I set in my vimrc, the old CS symbol is supported by my font and showed correctly, but not became a '?' symbol.

Reply:
if you say that this is no problem since vim-ctrlspace does not notice it anyhow, then it would be okay for me. We can change it later, if others complain :)

Reply:
going to merge it then. Thanks

Reply:
Thank your receiving pullrequest too.